tests/Unit/Scraper/Listeners/UpdateDatasetTest.php

```diff
@@ -63,10 +60,11 @@ public function whenDatasetExistsItShouldBeUpdated(): void
      */
     public function whenDatasetDoesNotExistAndTheDatasetsLimitHasNotBeenReachedItShouldBeSaved(): void
     {
-        factory(ScrapedDataset::class, UpdateDataset::DATASET_AMOUNT_LIMIT - 1)->create([
+        ScrapedDataset::factory()->count(UpdateDataset::DATASET_AMOUNT_LIMIT - 1)->create([
```

Comment:
Nice! I like the count usage here. Could you change in the other tests to use `factory->count($amount)` instead of use `factory($amount)` to be consistent?

Reply:
Changes have been at all places. Can you please which is missing?

Reply:
You can find it searching in this pr something like `factory(<something>)`